config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultImagePath            = "/usr/share/clear-containers/clear-containers.img"
 	defaultHypervisorPath       = "/usr/bin/qemu-lite-system-x86_64"
 	defaultProxyURL             = "unix:///run/cc-oci-runtime/proxy.sock"
+	defaultPauseBinPath         = "/tmp/bundles/pause_bundle/rootfs/bin/pause"
```

Comment:
@sameo Just to make sure I understand correctly. We want `/var/lib/clearcontainers/runtime/bundles/pause_bundle` as default, and for example, the config.toml file will provide `rootfs/bin/pause` as a relative path to add to this default one ?
You mentioned this as optional, but it has to be mandatory because in case of runtime implementation, we will use hyperstart, meaning we must provide a pause binary path for the pause container. Do you agree ?
As a global comment, I don't understand why we are not providing the full path here through the config.toml file. Since we have to know the full path, there is no point in having one part of the path hardcoded by default and another part provided by the config file.